src/web/pages/PowerBIReportsPage.tsx

```diff
@@ -0,0 +1,102 @@
+import SideBar from '../components/SideBar';
+import TopMenuBar from '../components/TopMenuBar';
+
+export function PowerBIReportsPage() {
+    return (
+        <div style={{ display: 'flex', flexDirection: 'column', height: '100vh', overflowX: 'hidden', backgroundColor: '#f4f6f8' }}>
```

Comment:
Why does this page have the TopMenuBar and SideBar explicitly called out? I didn't notice that in other pages.

Reply:
It is an example of a page using the top menu bar and sidebar, the sample page has that too.

Reply:
I will put the sidebar and topmenubar components on the App.tsx file.  This way it will be present throughout the app. I has initially added it there to test but it is redundant.

Reply:
I created this mainlayout component that I will add to app.tsx. Already tested locally. The sample page is not needed anymore. You cam expect a new commit